doc/version_history.rst

```diff
@@ -21,7 +35,7 @@ Requires:
 
 * ts_salobj 7
 * ts_idl 3.7
-* IDL file for ESS from ts_xml 15 (14 is OK if not using SiglentSsa3000xSpectrumAnalyzerDataClient)
```

Comment:
I think ts_xml 14 is OK if not using. If so, please fix this in 2 places.

If 15 is truly needed (even if not using your new client) then when did that occur? I don't see it listed in the version history. Perhaps update the version history with that info?